Module.yaml

```diff
@@ -0,0 +1,62 @@
+version: 1.0.0
+name: OPCUADiscovery
+description: OPC UA Discovery looks for all available nodes onn OPC UA Server and forwards the payload to next module.
+icon: https://icons-020-demo.s3.eu-central-1.amazonaws.com/mi_OPCUA_input.png
+type: process
```

Comment:
If this module subscribes to a server, why would it be of type process? If you build a pipeline with this module, do you MUST use another module before this one? If not, then it should be input module. If it is stand alone it seems that it does not need input right? If it does not need input from another module, it should be `input` type

Reply:
ok, then it's input